bin/appServiceConfigurableTest.sh

```diff
@@ -19,7 +19,7 @@ echo "Info: Initiating AppServiceConfigurable Test."
 
 echo "[info] ---------- use docker-compose run newman ----------"
 
-docker-compose run --rm postman run ${COLLECTION_PATH} \
+docker-compose -f ../docker-compose-test-tools.yml docker-compose-test-tools.yml run --rm postman run ${COLLECTION_PATH} \
```

Comment:
Not sure what is going on here with the duplication of `docker-compose-test-tools.yml`.  What is this doing?

Reply:
It's a typo. Thank you for catching it.

Fixed.